packages/hoppscotch-common/src/helpers/import-export/import/openapi.ts

```diff
@@ -974,14 +1004,25 @@ export const hoppOpenAPIImporter = (fileContents: string[]) =>
             const resultDoc = []
 
             for (const docObj of docArr) {
-              const validatedDoc = await dereferenceDocs(docObj)
-
-              resultDoc.push(validatedDoc)
+              try {
+                const validatedDoc = await dereferenceDocs(docObj)
+                resultDoc.push(validatedDoc)
+              } catch (error) {
+                console.warn(
+                  "Dereferencing failed, using original document:",
+                  error
+                )
+                // If dereferencing fails, use the original document
+                resultDoc.push(docObj)
+              }
```

Comment:
If a non dereferenced document gets into our pipeline, and it has unresolved refs, we probably won't be able to properly convert it into a request ?